### DISCUSSION TITLE
有沒有WSA+Alas的完整配置教學

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#1514] [Q&A] [answered]
如Title，有完整教學嗎
我看置頂issue有點混亂，不知道這到底是完成了還是還沒
scrcpy怎麼用好像網上也不太明確

**Comment by SaiCateDoan** [comment, marked as answer]
由于WSA 2204进行了一个比较大的更新，现在alas暂时不支持
WSA 2203 或更老版本，alas支持让碧蓝航线在后台运行，没有窗口显示，如果需要操作，则需要使用scrcpy或其他类似软件
总体来说暂时不建议使用WSA
